resources/views/admin/createuser.blade.php

```diff
@@ -0,0 +1,233 @@
+      <input id="tendn" type="text"  name="name" required>
+     <label for="email"><b>Email</b></label>
+      <input type="text" placeholder="Nhập Email" name="email" required>
+     <label id="email" for="psw"><b>Password</b></label>
+      <input type="password" placeholder="Nhập Mật Khẩu" name="psw" required>
+     <label for="psw-repeat"><b>Nhập Lại Mật Khẩu</b></label>
+      <input type="password" placeholder="Nhập Lại Mật Khẩu" name="psw-repeat" required>
+     <label for="email"><b>Address</b></label>
+      <input type="text"  name="Address" required>
+     <label for="email"><b>Facebook link</b></label>
+      <input type="text" placeholder="https://example.com" name="facebook" required>
+     <label for="email"><b>Youtuba</b></label>
+      <input type="text" placeholder="https://example.com" name="youtube" required>
+     <label for="email"><b>Description</b></label>
+      <input type="text"  name="description" required>
+     <label>
+        <input type="checkbox" checked="checked" name="remember" style="margin-bottom:15px"> Nhớ Đăng Nhập
+      </label>
+     <div class="clearfix container">
+        <button type="submit" class="signupbtn ">Save</button>
+        <button type="submit" class="signupbtn ">Reset</button>
+      </div>
+     </div>
+     </div>
+   </div>
+ 
+</body>
+<style>
+     @import url('https://fonts.googleapis.com/css2?family=Noto+Sans+JP&display=swap');
+* {box-sizing: border-box}
+body{
+  font-family: 'Noto Sans JP', sans-serif;
+}
+h1, label{
+  color: DodgerBlue;
+}
+h2{
+  padding-left: 16px;
+    margin: -4px 0 4px 0;
+    width: 204px;
+}
+.menu{
+    height: 100%;
+    width: 100%;
+    background-color: #E7E9EB;
+    
+    overflow-x: hidden;
+    padding-top: 20px;
+    display: block;
+    font-family: Verdana,sans-serif;
+    font-size: 15px;
+    line-height: 1.5;
+}
+/* Full-width input fields */
+  input[type=text], input[type=password] {
+  width: 75%;
+  padding: 15px;
+  margin: 5px 0 22px 0;
+  display: inline-block;
+  border: none;
+  width: 100%;
+  resize: vertical;
+  padding:15px;
+  border-radius:15px;
+  border:0;
+  box-shadow:4px 4px 10px rgba(0,0,0,0.2);
+}
+
+input[type=text]:focus, input[type=password]:focus {
+  outline: none;
+}
+
+hr {
+  border: 1px solid #f1f1f1;
+  margin-bottom: 25px;
+}
+
+/* Set a style for all buttons */
+button {
+  background-color: #4CAF50;
+  color: white;
+  padding: 14px 20px;
+  margin: 8px 0;
+  border: none;
+  cursor: pointer;
+  width: 25px;
+  opacity: 0.9;
+  height: 50px;
+}
+
+button:hover {
+  opacity:1;
+}
+a{
+  font-family: "Segoe UI",Arial,sans-serif;
+    text-decoration: none;
+    display: block;
+    padding: 2px 1px 1px 16px;
+    color: inherit;
+    background-color: transparent;
+    box-sizing: inherit;
+    
+}
+a:hover{
+  color: #000000;
+  background-color: #cccccc;
+}
+
+/* Extra styles for the cancel button */
+.cancelbtn {
+  padding: 14px 20px;
+  background-color: #f44336;
```

Comment:
Toàn bộ code css trong style đưa vào 1 file `app.css` nằm trong thư mục `public/css`